### PR TITLE
Update extensions to build 2017-06-13

### PR DESCRIPTION
**Pull request by ewpatton**
This PR includes updated versions of the extensions to address a number of bugs reported on the forum as well as to add a few new blocks to aid in getting information about the connected device and its services in a more App Inventor friendly way.

### CODE REVIEW

**Comment by ewpatton** [comment]
Do not merge yet. I am looking into two more possible bugs and will update this commit.

**Comment by ewpatton** [comment]
This is now the latest version of the extensions. It includes some additional bug fixes and adds the AutoReconnect and ConnectionTimeout properties to the BluetoothLE extension.

**Comment by ewpatton** [comment]
Do not merge (again). Some extensions need to have URLs fixed based on issue #134

**Comment by ewpatton** [comment]
Fixed links to PDFs in accel and gyro components of Arduino 101. Removed links for Arduino Pins since the documents do not exist.

**Comment by ewpatton** [comment]
This will need rebasing/updating after #137 is merged. Do not merge.

**Comment by ewpatton** [comment]
This should be the final version of extensions and documentation barring any requests for changes during review. @marksherman @afmckinney

File: docs/src/app/pages/MicroBitMagnetometer.md
Comment (marksherman) [inline docs/src/app/pages/MicroBitMagnetometer.md]: "toconfigure"

File: docs/src/app/pages/MicroBitUart.md
Comment (marksherman) [inline docs/src/app/pages/MicroBitUart.md]: Request -> Requests

File: docs/src/app/pages/MicroBitUart.md
Comment (marksherman) [inline docs/src/app/pages/MicroBitUart.md]: Stop -> Stops

File: docs/src/app/pages/Arduino101Pins.md
Comment (marksherman) [inline docs/src/app/pages/Arduino101Pins.md]: `-->` is rendering, along with the whole paragraph that you tried to comment out.

File: docs/src/app/pages/BluetoothLEIntro.md
Comment (marksherman) [inline docs/src/app/pages/BluetoothLEIntro.md]: no descriptive sentence?

File: docs/src/app/pages/BluetoothLEIntro.md
Comment (marksherman) [inline docs/src/app/pages/BluetoothLEIntro.md]: > The amount of time, in seconds, that the BluetoothLE component will wait for a connection to be established with a device when Connection or ConnectWithAddress.

when what?

Also, "Connection" and "ConnectWithAddress" are rendering differently. The first code tag is a close-tag.

File: docs/src/app/pages/BluetoothLEIntro.md
Comment (marksherman) [inline docs/src/app/pages/BluetoothLEIntro.md]: no descriptive sentence

File: docs/src/app/pages/BluetoothLEIntro.md
Comment (marksherman) [inline docs/src/app/pages/BluetoothLEIntro.md]: "Unique ID" and "UniqueID" both present in same sentence. Based on other entries the space should be used.

File: docs/src/app/pages/BluetoothLEIntro.md
Comment (marksherman) [inline docs/src/app/pages/BluetoothLEIntro.md]: should indeed be "Registers"

File: docs/src/app/pages/BluetoothLEIntro.md
Comment (marksherman) [inline docs/src/app/pages/BluetoothLEIntro.md]: Block image says "shortFloat" without the 's'. The description says "shortFloats" three times

File: docs/src/app/pages/BluetoothLEIntro.md
Comment (marksherman) [inline docs/src/app/pages/BluetoothLEIntro.md]: same thing: "shortFloat" in the block image but "shortFloats" in the text

File: docs/src/app/pages/BluetoothLEIntro.md
Comment (marksherman) [inline docs/src/app/pages/BluetoothLEIntro.md]: rendering as the beginning of a numbered list. Remove line break and it should be ok, I hope.

File: docs/src/app/pages/BluetoothLEIntro.md
Comment (marksherman) [inline docs/src/app/pages/BluetoothLEIntro.md]: codeify "signed"

File: docs/src/app/pages/BluetoothLEIntro.md
Comment (marksherman) [inline docs/src/app/pages/BluetoothLEIntro.md]: codeify "signed"

File: docs/src/app/pages/BluetoothLEIntro.md
Comment (marksherman) [inline docs/src/app/pages/BluetoothLEIntro.md]: codeify "signed"

File: docs/src/app/pages/BluetoothLEIntro.md
Comment (marksherman) [inline docs/src/app/pages/BluetoothLEIntro.md]: say true-case first (this bug may exist elsewhere, too)

File: docs/src/app/pages/BluetoothLEIntro.md
Comment (marksherman) [inline docs/src/app/pages/BluetoothLEIntro.md]: definitely exists elsewhere. say true case first

File: docs/src/app/pages/BluetoothLEIntro.md
Comment (marksherman) [inline docs/src/app/pages/BluetoothLEIntro.md]: is it shortFloats or shortFloat (with or without suffix s)

File: docs/src/app/pages/BluetoothLEIntro.md
Comment (marksherman) [inline docs/src/app/pages/BluetoothLEIntro.md]: sign or signed?

File: docs/src/app/pages/BluetoothLEIntro.md
Comment (marksherman) [inline docs/src/app/pages/BluetoothLEIntro.md]: sign or signed?

File: docs/src/app/pages/BluetoothLEIntro.md
Comment (marksherman) [inline docs/src/app/pages/BluetoothLEIntro.md]: say true first

File: docs/src/app/pages/MicroBitIoPin.md
Comment (marksherman) [inline docs/src/app/pages/MicroBitIoPin.md]: space between theI/O

File: docs/src/app/pages/MicroBitUart.md
Comment (marksherman) [inline docs/src/app/pages/MicroBitUart.md]: why mention users? Just say "provides" 
(Also, Hal came in today with a new mission that will be against the word "users")

File: docs/src/app/pages/MicroBitUart.md
Comment (marksherman) [inline docs/src/app/pages/MicroBitUart.md]: Writes

**Review by marksherman — CHANGES_REQUESTED** [review]

**Comment by ewpatton** [comment]
I've pushed updated documentation that should address these issues (and a few that weren't documented but of the same variety).

File: docs/src/app/pages/BluetoothLEIntro.md
Comment (marksherman) [inline docs/src/app/pages/BluetoothLEIntro.md]: UTF-9?

**Review by marksherman — CHANGES_REQUESTED** [review]
I love all your new intro wordings. Just one typo.

**Pull request merged**
